plat/rockchip/rk3399/aarch64/platform_common.c

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
Commit message could do with describing the SoC a bit more, at least the CPU topology.